projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -672,6 +679,46 @@ function ( $site_purchase ) {
 	return ! empty( $domain_purchases );
 }
 
+/**
+ * Check if the add_new_page task is complete.
+ *
+ * @return bool True if add_new_page task is completed.
+ */
+function wpcom_is_add_new_page_completed() {
+	return wpcom_is_task_option_completed( 'add_new_page' );
```

Comment:
The function `wpcom_is_task_option_completed` recieves an `array` as parameter, not a `string`. https://github.com/Automattic/jetpack/blob/ebf3671dc4f99b9fd0ce44955933fc249fef8e70/projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php#L484

Reply:
Whoops, I'll get that fixed. Thanks!